solara/settings.py

```diff
@@ -36,9 +38,13 @@ class Config:
         env_file = ".env"
 
 
+# in colab or vscode there is not solara cdn proxy available
+_should_not_use_proxy = solara.util.is_running_in_colab() or solara.util.is_running_in_vscode()
```

Comment:
If removing the double negative, this should be 
```suggestion
_should_use_proxy = not (solara.util.is_running_in_colab() or solara.util.is_running_in_vscode())
```